Auth0Tests/AuthenticationSpec.swift

```diff
@@ -808,6 +809,143 @@ class AuthenticationSpec: QuickSpec {
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, audience: nil, scope: nil, parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should not include audience by default") {
+                    stub(condition: isToken(Domain) && hasNoneOf(["audience"])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, scope: nil, parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should include scope if it is not nil") {
+                    stub(condition: isToken(Domain) && hasAtLeast(["scope": "scope"])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, audience: nil, scope: "scope", parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should not include scope if it is nil") {
+                    stub(condition: isToken(Domain) && hasNoneOf(["scope"])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, audience: nil, scope: nil, parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should use 'openid' as the default scope") {
+                    stub(condition: isToken(Domain) && hasAtLeast(["scope": "openid"])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, audience: nil, parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should return an id token when the scope includes 'openid'") {
```

Comment:
**Question:** Are these integration tests?

Reply:
You are right, that's an integration test. I'll remove it.

Reply:
Fixed in b718c8441f66d8259087aec2166fbac980cef0ce.